etc/onto-viewer-web-app/config/ontology_config.yaml

```diff
@@ -0,0 +1,14 @@
+ontologies:
+  source:
+    - url: 'https://spec.industrialontologies.org/ontology/AboutIOFDev/'
+  catalog_path:
+    - ontologies/catalog-v001.xml
```

Comment:
```suggestion
```

Not required after ['zip' set as 'URL#fragment' #121](https://github.com/edmcouncil/ontology-publisher/pull/121)

Reply:
Let's us leave it for now.